src/plugins/telemetry/public/services/telemetry_sender.ts

```diff
@@ -53,13 +53,22 @@ export class TelemetrySender {
     return false;
   };
 
+  private delayRetrySend = () => {
+    return 60 * (1000 * Math.min(Math.pow(2, this.retryCount), 64)); // 120s, 240s, 480s, 960s, 1920s, 3840s, 3840s, 3840s
+  };
+
   private sendIfDue = async (): Promise<void> => {
-    if (this.isSending || !this.shouldSendReport()) {
+    if (!this.shouldSendReport()) {
```

Comment:
Exit early if and only if the next report isn't due to be sent. This backs-off on the once-per minute fetch and send request calls we used to make.